src/java_bytecode/java_object_factory.cpp

```diff
@@ -181,10 +512,9 @@ Function: new_tmp_symbol
 
 \*******************************************************************/
 
-namespace {
-symbolt &new_tmp_symbol(symbol_tablet &symbol_table)
+symbolt &new_tmp_symbol(symbol_tablet &symbol_table, const std::string& prefix)
 {
-  static int temporary_counter=0;
+  static size_t temporary_counter=0; // TODO change this
```

Comment:
What should it be changed to? (At bare minimum make the comment more informative; even better: fix the problem right away.)